app/qtapp/rtkplot_qt/plotcmn.cpp

```diff
@@ -78,7 +78,12 @@ void Plot::showLegend(const QStringList &msgs)
 
     for (int i = 0; (i < 7) & (i < msgs.count()); i++) {
         lblQL[i]->setText(msgs[i]);
-        setWidgetTextColor(lblQL[i], plotOptDialog->getMarkerColor(sel, i + 1));
+        int col = i + 1;
```

Comment:
Could you add a comment here, describing the rational of the new code. I think without a comment it is hard to understand, once there is no relation to this PR anymore, what the code is for.

Reply:
The different color order for the observations counts gives the first marker colors to lower counts rather than  high counts. The default green color for two observations rather than for five. This suits the lower end receivers, giving a green (good) when they are receiving all their signals. But now there are common low end receiver data sets with more than 2 observations so this PR is to get the legend colors to match the plot colors for these.